packages/image_picker/image_picker/android/src/main/java/io/flutter/plugins/imagepicker/FileUtils.java

```diff
@@ -147,6 +42,7 @@ private static String getPathFromRemoteUri(final Context context, final Uri uri)
       String extension = getImageExtension(uri);
       inputStream = context.getContentResolver().openInputStream(uri);
       file = File.createTempFile("image_picker", extension, context.getCacheDir());
+      file.deleteOnExit();
```

Comment:
+1 Especially since these files can get large.

Although this would imply the apps should not be storing the File URI anywhere. I don't think we currently recommend that in the documentation. We should fix that.

Reply:
Good point, will update the doc as well!